sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/ConfluentSchemaRegistryDeserializerProviderTest.java

```diff
@@ -45,7 +45,8 @@ public void testGetCoder() {
     assertEquals(AVRO_SCHEMA, coderV0.getSchema());
 
     try {
-      Integer version = mockRegistryClient.register(subject, AVRO_SCHEMA_V1);
+      mockRegistryClient.register(subject, AVRO_SCHEMA_V1);
```

Comment:
How is this different? Or better how is this testing that the good schema is chosen? Maybe we can add a specific test for that?